frontend/src/app/chat/ChatHooks.js

```diff
@@ -132,4 +133,9 @@ export async function sendMessage(chatId, data) {
         release: data?.release,
         chat: data?.chat,
     })
+    saveEvent('send_chat_message', {
+        community: data?.community,
+        release: data?.release,
+        chat: data?.chat,
+    })
```

Comment:
Lauren will add more of these `saveEvent()` calls throughout the frontend to collect more analytics information.